rust.yaml

```diff
@@ -117,3 +117,31 @@ update:
   github:
     identifier: rust-lang/rust
     use-tag: true
+
+test:
+  environment:
+    contents:
+      packages:
+        - gcc
+        - glibc-dev
```

Comment:
should these be added as runtime dependencies?

Reply:
rustc needs any c compiler to be present 
https://users.rust-lang.org/t/why-rust-language-required-to-install-g-visual-c-gcc-additionally-for-compiling-rust-file/46123/15